mkosi/__init__.py

```diff
@@ -540,15 +540,19 @@ def format_bytes(num_bytes: int) -> str:
 _IOC_READ  = 2  # NOQA: E221
 
 
-def _IOC(dir_rw: int, type_drv: int, nr: int, argtype: str) -> int:
-    size = {"int": 4, "size_t": 8}[argtype]
+def _IOC(dir_rw: int, type_drv: int, nr: int, argtype: Optional[str] = None) -> int:
+    size = {"int": 4, "size_t": 8}[argtype] if argtype else 0
```

Comment:
Not that it matters, but one can lean into this being a dict :)
```suggestion
    size = {"int": 4, "size_t": 8}.get(argtype, 0)
```

Reply:
Unfortunately, mypy is not happy because argtype is optional

Reply:
Looks like it's python/mypy#9430. A workaround here would be to not change the type to `Optional[str]`, but keep it `str` and make the default the empty string. As I said, though, it doesn't really matter. I just saw the `[argtype]` on the dict literal and thought it cute it lean more heavily into this being a dict. :)

Reply:
Let's keep it like this for now I'd say